lib/fpm/package/deb.rb

```diff
@@ -905,7 +905,7 @@ def mkdir_p(dir)
   def to_s(format=nil)
     # Default format if nil
     # git_1.7.9.3-1_amd64.deb
-    return super("NAME_FULLVERSION_ARCH.TYPE") if format.nil?
+    return super("NAME_FULLVERSION_ARCH.EXTENSION") if format.nil?
```

Comment:
Another thought -- ``to_s`` is usually meant to represent object in a string form for displaying purposes. Here it looks like it has a purpose of generating output file name. Is it intentional? Shouldn't these two things be separated?